docs/user/troubleshooting.md

```diff
@@ -99,6 +99,230 @@ This is safe to ignore and merely indicates that the etcd bootstrapping is still
 
 The easiest way to get more debugging information from the installer is to check the log file (`.openshift_install.log`) in the install directory. Regardless of the logging level specified, the installer will write its logs in case they need to be inspected retroactively.
 
+### Installer fails to initialize the cluster
+
+The installer uses the [cluster-version-operator] to create all the components of an OpenShift cluster. When the installer fails to initialize the cluster, the most important information is looking at the [clusterversion] and [clusteroperator] objects:
+
+1. Inspecting the `clusterversion` object.
```

Comment:
Can we make these subsections with headers (e.g. `#### Inspecting the ClusterVersion object`)?  That way we get linkable anchors and pedantic folks don't worry about your subsequent fenced block falling outside of the list entry:

```console
$ curl -s https://github.com/openshift/installer/blob/dea429c16b29069ed6e1dffbc0d601b7edf88f32/docs/user/troubleshooting.md | grep -A2 'code>clusterversion</code'
<li>Inspecting the <code>clusterversion</code> object.</li>
</ol>
<div class="highlight highlight-text-shell-session"><pre>$ <span class="pl-s1">oc --config=<span class="pl-smi">${INSTALL_DIR}</span>/auth/kubeconfig get clusterversion -oyaml</span>
```

because it doesn't have a four-space indent ;)

Reply:
They supposed to be steps. so keeping it a list.